torchao/quantization/prototype/mixed_precision/scripts/FIT.py

```diff
@@ -0,0 +1,105 @@
+import torch
```

Comment:
nit: can you use lowercase names for these 3 files?

Reply:
I have renamed them